src/modules/login/components/Login.js

```diff
@@ -19,14 +21,21 @@ export const Login = () => {
         const password = data.get('password')
 
         try {
-            const res = await axios.post(
+            const response = await axios.post(
                 //TODO: update url once backend is finished
-                'https://zj7ipg4ixun5mj77y6g6t7qkce0omtjb.lambda-url.ap-southeast-2.on.aws/',
+                'http://localhost:3001/auth/login',
                 { email, password }
             )
 
-            if (res.data.isLoginSuccess) {
-                setIsLoading(false)
+            if (response.data.token) {
```

Comment:
could you add a TODO here. `// TODO:  backend will change to move user into data object`

Reply:
done